pylint/test/functional/stop_iteration_inside_generator.rc

```diff
@@ -0,0 +1,2 @@
+[testoptions]
+min_pyver=3.5
```

Comment:
I think an idea for this check is to have it enabled for all of 3.x branches. We want to indicate to users that it'll cause a problem in newer interpreter.

(But I'm not 100% sure).